rotkehlchen/db/evmtx.py

```diff
@@ -155,7 +155,7 @@ def get_evm_internal_transactions(
                 trace_id=result[0],
                 from_address=result[1],
                 to_address=result[2],
-                value=result[3],
+                value=int(result[3]),
```

Comment:
why this change here?

Reply:
So, this was because the value is typed as an int in the `EvmInternalTransaction`. But, here we get it from DB, so it's returned as str. This raises an error [here in the decoder](https://github.com/rotki/rotki/pull/7395/files#diff-dfc6191ba411ce3c637694024e930e2a9bb812dbbda6b3b7306c2adfe88f976aR144) FVal doesn't accept str.

Reply:
It was only used in one place and the test was expecting str instead of int https://github.com/rotki/rotki/blob/64beb7ec4c6e38f67d2d5f4c6444df88d525c1d2/rotkehlchen/tests/external_apis/test_etherscan.py#L185

I believe it was always executing the else branch of this condition https://github.com/rotki/rotki/blob/64beb7ec4c6e38f67d2d5f4c6444df88d525c1d2/rotkehlchen/chain/evm/decoding/decoder.py#L664